lib/combo.coffee

```diff
@@ -455,7 +461,13 @@
       if @emptyFieldValidation
         @input.toggleClass('empty', @getRawValue() is "")
 
-      htmls = []
+      htmls = [];
+
+      if @showUnmatchedRawValue

```

Comment:
Kan du ikke skrive

if @show... and !hasSelection and @getRawValue !== ""
  htmls.push('<li....)